Dockerfile

```diff
@@ -71,6 +71,10 @@ FROM python:3.10-alpine AS isso
 WORKDIR /isso/
 COPY --from=isso-builder /isso/ .
 
+LABEL org.opencontainers.image.source=https://github.com/isso-comments/isso
```

Comment:
This isn't mentioned in the pr description

Reply:
Amended the PR description